### ISSUE TITLE
Composer install fails

### ISSUE THREAD

**Issue by jameswithers** [magento/magento2#2612]
```
[root@demo html]# composer create-project --repository-url=https://repo.magento.com/ magento/project-community-edition .
Installing magento/project-community-edition (2.0.0)
  - Installing magento/project-community-edition (2.0.0)
    Downloading: 100%



  [RuntimeException]
  Could not delete ./:
```

**Comment by jameswithers** [comment]
Problem was not all PHP required extensions had been installed.

**Comment by dariolongo** [comment]
Hi,
I've the same problem.
I've created a new user and added it to web server group.
```
magento_user : magento_user www-data
```

From this user, I run: composer create-project --repository-url=https://repo.magento.com/ magento/project-community-edition .
in the default /var/www/html directory

```
Installing magento/project-comunity-edition (2.2.2)
     -  Installing magento/project-comunity-edition (2.2.2): Downloading (100%)
Downloading (100%)
Downloading (100%)

[RuntimeException]
Could not delete ./:
```

I have checked the all php extension (php -me) and they are the same of [http://devdocs.magento.com/guides/v2.2/install-gde/prereq/php-ubuntu.html](http://devdocs.magento.com/guides/v2.2/install-gde/prereq/php-ubuntu.html)

I've changed all configuration in php.ini for opcache also.

php 7.1.14
apache 2.4.27
mysql 5.7.20

Did someone fix this?
Thanks?
